pkannan3/guide-me
# Landing page

Changes:

1. Made entire landing page responsive in size.
2. Added down arrow and added functionality.
3. Added developer cards and functionality.


Test:

1. Adjust the size of the browser window to confirm the content stays centered and is responsive in size.
5. Confirm down arrow is pulsing.
6. Hover over down arrow, it should stop pulsing and change in font-weight.
7. Select down arrow, should scroll down to developer cards.
8. Hover over developer cards, they should be responsive in size.
9. Hover over buttons within the developer cards. They should be responsive in size and font-weight.
